lib/controllers/src/OpenLoop.cpp

```diff
@@ -45,9 +48,8 @@ bool OpenLoop::shouldStop() {
                     std::abs(speed) > m_maxDegPerSec;
   shouldStop = shouldStop || (!m_ignoreCurrentChecks.load() && m_currentSensor &&
                               m_currentSensor.value().get().read(current) && std::abs(current) > m_maxCurrent);
-  shouldStop = shouldStop || (m_upperLimit.is_connected() && m_upperLimit.read() && m_setpoint.load() > 0);
-  shouldStop = shouldStop || (m_lowerLimit.is_connected() && m_lowerLimit.read() && m_setpoint.load() < 0);
-
+  shouldStop = shouldStop || (m_upperLimit.is_connected() && !m_upperLimit.read() && m_setpoint.load() > 0);
+  shouldStop = shouldStop || (m_lowerLimit.is_connected() && !m_lowerLimit.read() && m_setpoint.load() < 0);
```

Comment:
limit switches are active low? We should have some abstraction for them, in a future task